docs/project/development.md

```diff
@@ -376,135 +239,84 @@ $ make release
 
 The binary will be located at `packages/bun-{platform}-{arch}/bun`.
 
-## Valgrind
-
-On Linux, valgrind can help find memory issues.
-
-Keep in mind:
+## Valgrind (Linux Only)
 
-- JavaScriptCore doesn't support valgrind. It will report spurious errors.
-- Valgrind is slow
-- Mimalloc will sometimes cause spurious errors when debug build is enabled
+On Linux, Valgrind can help find memory issues. Keep in mind:
 
-You'll need a very recent version of Valgrind due to DWARF 5 debug symbols. You may need to manually compile Valgrind instead of using it from your Linux package manager.
+- JavaScriptCore doesn't support Valgrind and may report spurious errors.
+- Valgrind is slow.
+- Mimalloc may cause spurious errors when the debug build is enabled.
 
-`--fair-sched=try` is necessary if running multithreaded code in Bun (such as the bundler). Otherwise it will hang.
+Ensure you have a recent version of Valgrind with DWARF 5 debug symbols. You may need to compile Valgrind manually:
 
 ```bash
 $ valgrind --fair-sched=try --track-origins=yes bun-debug <args>
 ```
 
-## Updating `WebKit`
+## Updating WebKit
 
-The Bun team will occasionally bump the version of WebKit used in Bun. When this happens, you may see something like this with you run `git status`.
-
-```bash
-$ git status
-On branch my-branch
-Changes not staged for commit:
-  (use "git add <file>..." to update what will be committed)
-  (use "git restore <file>..." to discard changes in working directory)
-	modified:   src/bun.js/WebKit (new commits)
-```
-
-For performance reasons, `make submodule` does not automatically update the WebKit submodule. To update, run the following commands from the root of the Bun repo:
+The Bun team may update the version of WebKit used in Bun. When this happens, you might see changes in the WebKit submodule. To update it, run the following commands from the Bun repository root:
 
 ```bash
 $ bun install
 $ make cpp
 ```
 
-<!-- Check the [Bun repo](https://github.com/oven-sh/bun/tree/main/src/bun.js) to get the hash of the commit of WebKit is currently being used.
-
-{% image width="270" src="https://github.com/oven-sh/bun/assets/3084745/51730b73-89ef-4358-9a41-9563a60a54be" /%} -->
-
-<!--
-```bash
-$ cd src/bun.js/WebKit
-$ git fetch
-$ git checkout <hash>
-``` -->
+If you encounter issues, check the [Bun repo](https://github.com/oven-sh/bun/tree/main/src/bun.js) for the hash of the current WebKit commit in use.
 
 ## Troubleshooting
 
-### 'span' file not found on Ubuntu
+If you encounter issues during the setup process, here are some troubleshooting steps:
```

Comment:
i dont think this line is needed, and it also implies these are only setup issues. i think that *is* the case right now but we may add some more of these in the future.